app/controllers/admin/merchants_controller.rb

```diff
@@ -10,6 +10,21 @@ def index
   def show
   end
 
+  def new
+    @merchant = Merchant.new
+  end
+
+  def create
+    @merchant = Merchant.new(merchant_params)
+    if @merchant.save
```

Comment:
nice sad path handling here, we'll want to implement the same on the merchant_item side of things